### ISSUE TITLE
[Feature]: Have the ability to show multiple dates ?

### ISSUE THREAD

**Issue by mathisgauthey** [mgmeyers/obsidian-kanban#858]
### Goal or desired outcome of this feature

![image](https://github.com/mgmeyers/obsidian-kanban/assets/46576952/52c62fbb-f0a5-4782-92d7-25b332cc2b4a)

### Describe the feature

I'd like to be able to have two different dates on tasks : 
- A start date
- A done date

Basically, having access to multiple date would allow one to use dates the way he wants.

In my case this is for a media backlog (games, manga, series and stuff). But it might be project management with start date, scheduled date, due date, done date.

### Can you think of any alternatives or work-arounds?

For now, just being able to hard-type two dates like @{2023-06-04} @{2023-11-18} and showing them both with a custom CSS would be enough. But I don't have the right knowledge to do that yet so I hope someone did it before and might want to share it !

### Screenshots, mockups, or videos

There's one up there.

**Comment by MicaelJarniac** [comment]
It seems to be possible to type multiple dates, but it's glitchy. For me, clicking on any of the dates to edit it will result in the first one being edited.

![image](https://github.com/mgmeyers/obsidian-kanban/assets/19514231/38e19d22-3be5-485f-991b-b8af6da1d208)

**Comment by jcesguerram** [comment]
yes, the possibility of having a start date and an end date would be great!
